### PR TITLE
Feature/#4/multilingual

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Added multilingual feature and please review it

### CODE REVIEW

**Review by wildone — APPROVED** [review]

**Review by wildone — CHANGES_REQUESTED** [review]
can we please do this PR into Develop branch.

**Comment by Rahul139-stack** [comment]
> can we please do this PR into Develop branch.
Base  branch changed to develop

File: cypress.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "baseUrl": "http://localhost:5050"
+  "baseUrl": "http://localhost:3000"
```
[the remark this answers is not in the document]
Comment (Rahul139-stack) [reply cypress.json:2]: #32 ticket raised for the same

**Comment by Rahul139-stack** [comment]
Reopen PR

**Review by ankitvats1 — APPROVED** [review]

**Review by wildone — APPROVED** [review]

**Pull request merged**